backend/infrahub/core/attribute.py

```diff
@@ -524,11 +525,18 @@ async def to_graphql(
         for field_name in field_names:
             if field_name == "updated_at":
                 updated_at = self._get_updated_at()
-                response[field_name] = updated_at.to_graphql() if updated_at else None
+                response[field_name] = await updated_at.to_graphql() if updated_at else None
                 continue
 
             if field_name == "updated_by":
-                response[field_name] = self._get_updated_by()
+                response[field_name] = (
+                    {
+                        "id": self._get_updated_by(),
+                        "__kind__": InfrahubKind.ACCOUNT,
+                    }
+                    if self._get_updated_by()
+                    else None
+                )
```

Comment:
We want this to be a CoreAccount so we can't just return a string, will need to look at some resolver to grab the display label and such as a follow up.